spec/factories/article.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :article do
+    title "Article Title"
+    body "This is the first official Article of FantasyMetrix."
+    user_id 1
```

Comment:
Try not to hardcode ids in factories. As the database is cleared out between tests, there's no guarantee that the User with id 1 will be in the users table.

Favor using the associations in factories, try:

```diff
- user_id 1
+ user
```